### DISCUSSION TITLE
Map right command key on macos

### DISCUSSION THREAD

**Discussion by rawsp33d** [kmonad/kmonad#599] [Q&A] [answered]
Is there currently a way to map the right command key on mac? Either in `master` or `keycode-refactor`.

I don't have much experience on macos and don't even really know what the key is canonically called, like how `super` is often referred to as `meta` on Linux, etc. Thus I'm finding it hard to even search for.

I want to map the right command key to send some other keycode, like hyper or some other prefix that works on mac (hopefully), so that I can recreate my Linux setup where right alt is my main super key (`ralt => lmet`).

**Comment by MuhammedZakir** [comment, marked as answer]
It's `rmet`.

See this (incomplete) doc: https://github.com/kmonad/kmonad/blob/keycode-refactor/doc/keycode_reference.org.

**Reply by rawsp33d** [reply]
That page is incredibly helpful, thank you! `rmet` worked perfectly.
